pallas-txbuilder/src/conway.rs

```diff
@@ -230,7 +230,8 @@ impl BuildConway for StagingTransaction {
                 language_view,
             };
 
-            dbg!(&dta);
+            // if we need to debug then uncomment
+            // dbg!(&dta);
```

Comment:
you can safely remove that, I was meaning to remove it myself (was left there unintentionally)

Reply:
removed